screens/FavPokemon.tsx

```diff
@@ -1,52 +1,28 @@
-import React, { useEffect, useState } from 'react';
-import { StyleSheet, Text, View, Image, Button } from 'react-native';
-import getFavPokemon from '../api/pokemon';
-import styles from '../style/base';
-import PropRow from '../components/PropRow';
+import React from 'react';
+import { createStackNavigator } from '@react-navigation/stack';
+import FavMain from './FavMain';
+import FavMoreInfo from './FavMoreInfo';
 
-const FavPokemon = ({ navigation }) => {
-  const [favPokemon, setFavPokemon] = useState(null);
-  const [loading, setLoading] = useState(true);
+const Stack = createStackNavigator();
 
-  useEffect(() => {
-    async function effect() {
-      const pokemon = await getFavPokemon('vulpix');
-      setFavPokemon(pokemon);
-      setLoading(false);
-    }
-    effect();
-  }, []);
+enum StackScreenNames {
+  FAV_MAIN = 'FavMain',
```

Comment:
`FAV_MAIN` (here and in the file/function name) is fine, but we usually call the first screen in a stack `root` or we avoid adding navigation-related naming to screens at all to keep them reusable.

I think it would be a bit better to call this file (Stack) `FavPokemonStack` and the screens `FavPokemon`/`FavMoreInfo`